crates/swc_ecma_quote/Cargo.toml

```diff
@@ -16,4 +16,6 @@ swc_common = {version = "0.17.9", path = "../swc_common"}
 swc_ecma_ast = {version = "0.68.3", path = "../swc_ecma_ast"}
 swc_ecma_quote_macros = {version = "0.1.0", path = "../swc_ecma_quote_macros"}
 swc_ecma_utils = {version = "0.69.0", path = "../swc_ecma_utils"}
-testing = {version = "0.18.0", path = "../testing"}
+
+[dev-dependencies]
+testing = {version = "0.18.0", path = "../testing"}
```

Comment:
I think it is ok to let this be devdeps?

Reply:
Yeah you are right.